byteps/tensorflow/__init__.py

```diff
@@ -29,6 +29,7 @@
 from byteps.tensorflow.ops import size, local_size, rank, local_rank
 from byteps.tensorflow.ops import handle_average_backwards_compatibility
 from byteps.tensorflow.util import _executing_eagerly
+from . import distribute
```

Comment:
Do we have to always import ``distribute`` ?

Reply:
maybe not. we can import it directly in the user program.